ui/nuxeo-user-group-management/nuxeo-user-management.js

```diff
@@ -496,6 +503,13 @@ import '../nuxeo-button-styles.js';
         _currentUser: {
           type: Object,
         },
+
+        _headers: {
+          type: String,
+          value() {
+            return JSON.parse(config.get('user.management.headers', {}));
```

Comment:
I believe that we should handle this in a similar way to the ´fetch´ property, as ideally, we should be able to contribute to this from different sources (see [code](https://github.com/nuxeo/nuxeo-web-ui/blob/maintenance-3.0.x/elements/nuxeo-app.js#L1520)).